include/envoy/access_log/access_log.h

```diff
@@ -49,6 +50,14 @@ class AccessLogManager {
    * @return the opened file.
    */
   virtual AccessLogFileSharedPtr createAccessLog(const std::string& file_name) PURE;
+
+  /**
+   * Create a new access log file managed by the access log manager.
+   * @param file_info specifies the file to create/open.
+   * @return the opened file.
+   */
+  virtual AccessLogFileSharedPtr
+  createAccessLog(const Envoy::Filesystem::FilePathAndType& file_info) PURE;
```

Comment:
Can we just merger this into the previous interface function? Or split into file version and stderr/stdout version? It seems strange to have both.

Reply:
I will make `  virtual AccessLogFileSharedPtr createAccessLog(const Envoy::Filesystem::FilePathAndType& file_info) PURE;` be the only interface. I would to change all the callers to use this interface.

Reply:
fixed in 7b221f7aabfb02994e96528dfa648421e1f54a7b